habitat-lab/habitat/gui/text_drawer.py

```diff
@@ -62,7 +63,7 @@ def __init__(
         self._max_display_text_chars = max_display_text_chars
 
         # Glyphs we need to render everything
-        self._glyph_cache = text.GlyphCache(mn.Vector2i(256))
+        self._glyph_cache = text.GlyphCache(mn.Vector2i(6 * 256))
```

Comment:
How about a comment on how we're choosing this number?

Reply:
Hmm, and with https://github.com/facebookresearch/habitat-sim/pull/2243 now in, which includes the above-mentioned better atlas packer, it probably doesn't have to be that large anymore.

Reply:
Just tested, still has to be at least 6 * 256, otherwise text don't appear ob the screen.